tokio/Cargo.toml

```diff
@@ -84,6 +84,11 @@ signal = [
 sync = []
 test-util = ["rt", "sync", "time"]
 time = []
+
+# Internal test-only feature. Not part of public API. Subject to change without notice.
+# Used for internal testing infrastructure.
+__internal_test = []
```

Comment:
nit: I was thinking about `integration_test` as a name of this feature. IMO it makes it more clearer that the functionality behind it is used for the IT tests.